secp256k1-zkp-sys/src/zkp.rs

```diff
@@ -363,6 +373,28 @@ extern "C" {
         sig: *const WhitelistSignature,
     ) -> c_int;
 
+    #[cfg_attr(
+        not(feature = "external-symbols"),
+        link_name = "rustsecp256k1zkp_v0_8_0_musig_pubkey_ec_tweak_add"
+    )]
+    pub fn secp256k1_musig_pubkey_ec_tweak_add(
+        cx: *const Context,
+        output_pubkey: *mut PublicKey,
+        keyagg_cache: *mut MusigKeyAggCache,
+        tweak32: *const c_uchar,
+    ) -> c_int;
+
+    #[cfg_attr(
+        not(feature = "external-symbols"),
+        link_name = "rustsecp256k1zkp_v0_8_0_musig_pubkey_xonly_tweak_add"
+    )]
+    pub fn secp256k1_musig_pubkey_xonly_tweak_add(
+        cx: *const Context,
+        output_pubkey: *mut XOnlyPublicKey,
```

Comment:
`secp256k1_musig_pubkey_xonly_tweak_add` expects a `PublicKey`, not `XOnlyPublicKey`.
This is causing signature verification to fail.

```suggestion
        output_pubkey: *mut PublicKey,
```

Reply:
Good catch.